### ISSUE TITLE
`hvm use --useVersionInDotFile` should work independently from `numtagstodisplay` config option

### ISSUE THREAD

**Issue by salim-b** [jmooring/hvm#44]
Currently, `hvm use --useVersionInDotFile` is only able to install versions that fall into the range of the `numtagstodisplay` config option.

If I set `numtagstodisplay = 6` and then try to `hvm use --useVersionInDotFile` on a `.hvm` file that contains `v0.116.0`, I get

> Error: the version specified in the .hvm file (v0.116.0) is not available in the repository: run "hvm use" to select a version, or "hvm disable" to remove the file

With `numtagstodisplay = 7`, the above Hugo version installs fine (until the next Hugo version is released).

**Comment by jmooring** [comment]
@salim-b Good catch. Please pull from source and continue to hammer on it as time allows. I'll roll another release on Wednesday if no more issues arise.
